demo/src/routes/docs/[framework]/getting-started/installation/installation.md

```diff
@@ -0,0 +1,30 @@
+# Installation
+
+## Prerequisites
+
+Before you begin, make sure you have the following prerequisites installed on your system:
+
+- Node.js: You can download and install Node.js from the [official website](https://nodejs.org/).
+
+## Create a new project
+
+If you haven't already, create a new project where you want to use AgnosUI. You can use the `blabla` for this.
+
+## Install AgnosUI
+
+AgnosUI provides two flavours of the widgets for you to choose from:
+
+- **Headless**: The {$selectedFramework$}-headless package contains the headless functionality of the widget library. It can be used with any front-end CSS framework. It uses the core of AgnosUI which is framework agnostic.
```

Comment:
If we use variables in the MD like `{$selectedFramework$}`, I think you will need a custom renderer else it will be rendered as html